packages/devtools_app/analysis_options.yaml

```diff
@@ -5,7 +5,7 @@ analyzer:
     - build/**
     - lib/generated_plugin_registrant.dart
     - macos/
-# Enable this option if you need to test whether the inspector_polyfill_script is compatible with
-# versions of flutter that require null safety.
-#  enable-experiment:
-#    - non-nullable
+
+linter:
+  rules:
+    - sort_child_properties_last
```

Comment:
is there any reason this shouldn't be where all our other lint rules are defined (../../analysis_options.yaml) ?

Reply:
User error. Fixed.